### PR TITLE
[KC-2017] Added KC for 2017

### PR DESCRIPTION
**Pull request by barkerd427**
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2017] Add Bluth Company as a sponsor`*

### CODE REVIEW

**Comment by MikeRosTX** [comment]
Please remove the redirect to Eventbrite for **welcome** on the navbar since it doesn't exist. It will be very confusing to your audience.

File: content/events/2017-kansascity/sponsor.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
++++
+<div style="width: 100%">
+  <div style="float:left; width: 50%;">
+    <img alt="DevOpsDays Kansas City 2017" src="/events/2017-kansascity/logo.png" style="max-width: 90%;">
+  </div>
+
+  <div style="float:left; width: 50%; padding-top: 12%;">
+We greatly value sponsors for this open event.  If you are interested in sponsoring, please drop us an email at {{< email_organizers >}}.
+
+Download the full <a href="/events/2016-kansascity/SponsorshipProspectusv2016.pdf" target="_blank">sponsor prospectus</a> for devopsdays Kansas City 2017!
```
Comment (aaronblythe) [inline content/events/2017-kansascity/sponsor.md]: Maybe not link to this until the 2017 is ready?  Or upload a draft of the 2017 and update over time?

File: content/events/2017-kansascity/welcome.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
++++
+date = "2016-05-07T09:30:25-05:00"
```
Comment (aaronblythe) [inline content/events/2017-kansascity/welcome.md]: I assume these were copied/pasted based on these dates.  I usually update them to the date manually.  Over time I would like to learn a better way to update these dates (or omit them if the git history is enough).

File: content/events/2017-kansascity/registration.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+type = "event"
+
+
++++
+
+<div style="width:100%; text-align:left;">
+
+<a href="http://devopsdayskc.eventbrite.com/?aff=website"> Registration site </a>
+
+<div style="width:100%; text-align:left;" ><iframe  src="https://www.eventbrite.com/e/devopsdays-kansas-city-2016-tickets-25170130538?ref=eweb" frameborder="0" height="1000" width="100%" vspace="0" hspace="0" marginheight="5" marginwidth="5" scrolling="auto" allowtransparency="true"></iframe><div style="font-family:Helvetica, Arial; font-size:10px; padding:5px 0 5px; margin:2px; width:100%; text-align:left;" ><a class="powered-by-eb" style="color: #dddddd; text-decoration: none;" target="_blank" href="http://devopsdayskc.eventbrite.com/?aff=website">Powered by Eventbrite</a></div></div>
```
Comment (aaronblythe) [inline content/events/2017-kansascity/registration.md]: Last year this caused issues in the way this page was taken out and readded.  I think it is worth setting up the EventBrite now and simple expose a coming soon type of display then we can update over time.

Comment (barkerd427) [inline content/events/2017-kansascity/registration.md]: I agree, but I thought I removed everything about registration. Creating it now.

File: data/events/2017-kansascity.yml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+coordinates: "39.082920, -94.582451" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+#location: "Musical Theater Heritage" # Defaults to city, but you can make it the venue name.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+  - name: welcome
+#  - name: program
+#  - name: speakers
+#  - name: location
+#  - name: registration
+    url: http://devopsdayskc.eventbrite.com/?aff=website
```
Comment (MikeRosTX) [inline data/events/2017-kansascity.yml]: Remark out **L26** of **2017-kansascity.yml**.

**Comment by barkerd427** [comment]
I've added the eventbrite page which should address the comments.

File: content/events/2017-kansascity/welcome.md (hunk 2 above)
Comment (barkerd427) [inline content/events/2017-kansascity/welcome.md]: I never pay attention to them.

File: content/events/2017-kansascity/sponsor.md (hunk 1 above)
Comment (barkerd427) [inline content/events/2017-kansascity/sponsor.md]: Oh, I just hid the page from the nav bar.

**Pull request merged**